Makefile

```diff
@@ -0,0 +1,25 @@
+# A Self-Documenting Makefile: http://marmelab.com/blog/2016/02/29/auto-documented-makefile.html
+
+SOURCE_FILES?=./...
+COVERAGE=coverage.out
+
+export PATH := ./bin:$(PATH)
+
+.PHONY: fmt
+fmt: ## Format code
+	@echo "==> Formatting all files..."
+	find . -name '*.go' | while read -r file; do gofmt -w -s "$$file"; goimports -w "$$file"; done
+
+.PHONY: test
+test: ## Run tests
```

Comment:
Seems easier to just run from your command line `go test -race ./...` and add coverage if you want.